backend/src/test/scala/cromwell/backend/validation/RuntimeAttributesValidationSpec.scala

```diff
@@ -329,13 +330,17 @@ class RuntimeAttributesValidationSpec extends WordSpecLike with Matchers with Be
         s"""
            |  default-runtime-attributes {
            |     memory: "2 GB"
+           |     memoryMin: "2 GB"
+           |     memoryMax: "2 GB"
```

Comment:
Maybe a little variety here to make each field is being wired through to the right place? eg `memoryMax: "3 GB"`